lib/rules/cli/kubectl-4.1.yaml

```diff
@@ -145,6 +145,17 @@
     :name: <value>
     :from_file: --from-file=<value>
     :from_literal: --from-literal=<value>
+:create_deployment:
+  :cmd: kubectl create deployment <name>
+  :options:
+    :allow_missing_template_keys: --allow-missing-template-keys=<value>
+    :dry_run: --dry-run=<value>
+    :generator: --generator=<value>
+    :image: --image=<value>
+    :output: --output <value>
```

Comment:
@kasturinarra should it be `--output=<value>`?

Reply:
@pruan-rht I took create deloyment which already exist in oc-4.1.yaml and did similar one for kubectl, i think this should be fine.

Reply:
@kasturinarra your assumption is that the 4.1 yaml is correct in the first place.  Although it works, but in reality if you look at the help file it shows that a `=` is present like other commands.  

> Create a deployment with the specified name.
> 
> Aliases:
> deployment, deploy
> 
> Usage:
>   oc create deployment NAME --image=image [--dry-run] [flags]
> 
> Examples:
>   # Create a new deployment named my-dep that runs the busybox image.
>   oc create deployment my-dep --image=busybox
> 
> Options:
>       --allow-missing-template-keys=true: If true, ignore any errors in templates when a field or map key is missing in
> the template. Only applies to golang and jsonpath output formats.
>       --dry-run=false: If true, only print the object that would be sent, without sending it.
>       --generator='': The name of the API generator to use.
>       --image=[]: Image name to run.
>   -o, --output='': Output format. One of:
> json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-file.
>       --save-config=false: If true, the configuration of current object will be saved in its annotation. Otherwise, the
> annotation will be unchanged. This flag is useful when you want to perform kubectl apply on this object in the future.
>       --template='': Template string or path to template file to use when -o=go-template, -o=go-template-file. The
> template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].
>       --validate=false: If true, use a schema to validate the input before sending it

Reply:
@kasturinarra I would suggest to add the `=` and correct it going forward to be consistent with the other options which all have the `=`.

Reply:
@pruan-rht changed as per your comments here, please help review, thanks !!